apps/backend/cypress/e2e/project.cy.ts

```diff
@@ -1,77 +1,96 @@
     draft: true,
-    datasets: 'tutu',
-    owner: 'possesseur',
-    contributors: 'contributeurs',
-    config: 'config',
-    version: 'versionNumber',
-    createdAt: 'dateCreation',
-  };
-  const modified_test_project = {
-    title: 'modified_project_test',
-    description: 'modified_project_test_description',
-    draft: true,
-    datasets: 'toto',
-    owner: 'possesseur',
-    contributors: 'contributeurs',
-    config: 'config',
-    version: 'versionNumber',
-    createdAt: 'dateCreation',
-  };
-  let id_test_project = null;
-  it('Project -> creation of new project (should return new project id)', () => {
+    datasets: { toto: 'test 2' },
+    description: 'description du projet 2',
+    config: { toto: 'test 2' },
+    version: 'version test',
+    contributors: [63, 64], // Shall we really send IDs ? Maybe usernames instead ?
+  });
+  const test_project_modified = new ProjectDto({
+    title: 'titre projet test 2_',
+    draft: false,
+    datasets: { toto: 'test 2_' },
+    description: 'description du projet 2_',
+    config: { toto: 'test 2_' },
+    version: 'version test_',
+    contributors: [63], // Shall we really send IDs ? Maybe usernames instead ?
+  });
+  let jwtUserAdmin;
+  let idUserAdmin;
+  it('Auth -> Connecting correctly with admin user.', () => {
     cy.request({
       method: 'POST',
-      url: '/api/project',
+      url: '/api/auth/login',
+      body: {
+        username: 'admin',
+        password: 'admin',
+      },
+      failOnStatusCode: false,
+    }).then((response) => {
+      jwtUserAdmin = response.body.access_token;
+      idUserAdmin = response.body.user_id;
+      test_project.owner = idUserAdmin;
+      expect(response.status).to.eq(201);
+    });
+  });
+  it('Project -> creation of new project -> should not fail.', () => {
+    cy.request({
+      method: 'POST',
+      url: '/api/projects',
       body: test_project,
+      auth: {
+        bearer: jwtUserAdmin,
+      },
       failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(201);
-      expect(response.body.id).to.be.a('number').greaterThan(0);
-      id_test_project = response.body.id;
     });
   });
-  it('Project -> Get info about project using id', () => {
+  it('Project -> get all projects -> should not fail.', () => {
     cy.request({
       method: 'GET',
-      url: '/api/project/' + id_test_project,
+      url: '/api/projects',
+      auth: {
+        bearer: jwtUserAdmin,
+      },
       failOnStatusCode: false,
     }).then((response) => {
       expect(response.status).to.eq(200);
-      // should return a object with the project info
-      expect(response.body).to.be.an('object');
+      expect(response.body).to.be.an('array');
     });
   });
-  it('Project -> Get all projects', () => {
+  it('Project -> Get info about project using id', () => {
     cy.request({
       method: 'GET',
-      url: '/api/projects',
+      url: '/api/projects/' + 1,
```

Comment:
```suggestion
      url: `/api/projects/${String(1)}`,
```
Prefer template literals for concatenation.